R/getMappableRegions.R

```diff
@@ -0,0 +1,189 @@
+#' @title Calc mappability of a genome.
+#'
+#' @description Given a window width and the maximum number of allowed hits per
```

Comment:
Given a k-mer length and the maximum number of allowed hits per k-mer, find all mappable regions in a genome.